docs/source/pages/operative_temperature_range.rst

```diff
@@ -0,0 +1,81 @@
+.. _operative_temperature_range:
+
+Operative temperature range
+===========================
+
+VPU temperature range
+*********************
+
+The visual processing unit - VPU (MyriadX) - is rated for industrial use and has operating 
+temperature -40 °C to 105 °C as seen `here <https://ark.intel.com/content/www/us/en/ark/products/125926/intel-movidius-myriad-x-vision-processing-unit-4gb.html#tab-blade-1-0-4>`__.
+Other components have higher max temperature, so 105 °C is the maximum working temperature for our devices.
+
+General use case
+****************
+
+In the general use case at a 25 °C ambient, the chip running at full power (i.e. worst-case heat generation) resides at around 70 °C maximum, 
+which sets the thermal margin to a minimum of 35 °C.
+
+Since devices generate about 45 °C in worst-case, this means the maximum ambient temperature should *theoretically* be:
+:code:`105 °C - 45 °C = 60 °C`.
+
+Command used for testing
+************************
+
+We used the following command when thermal testing our devices:
+:code:`python3 test.py -nce 2 -sh 10 -cmx 10 -rgbf 30 -rgbr 2160 -s metaout previewout meta_d2h -v /dev/null`
+
+Since this is Gen1 software, the equivalent Gen2 command should be:
+:code:`python3 depthai_demo.py -sub -lrc -hq -rgbr 2160 -v /dev/null --report cpu`
```

Comment:
let's just say we have tested thermals of our device with max load, and you can achieve that by using the depthai_demo. And add encoding argument here. Check depthai repo on how to do that.